docs/content/en/docs/getting-started/local-environment/_index.md

```diff
@@ -70,46 +70,116 @@ To install the EKS Anywhere binaries and see system requirements please follow t
-   The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription.
-   ------------------------------------------------------------------------------------------------------------------------------
-   Installing helm chart on cluster	{"chart": "eks-anywhere-packages", "version": "0.2.0-eks-a-v0.0.0-dev-build.3842"}
-   secret/aws-secret created
-   job.batch/eksa-auth-refresher created
-   package.packages.eks.amazonaws.com/my-harbor created
-   ```
+2. Create Cluster: Create your cluster either with or without curated packages:
+
+   - Cluster creation without curated packages installation
+
+     *Note* The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. Due to this there might be some warnings in the CLI if proper authentication is not set up.
+      ```bash
+      eksctl anywhere create cluster -f $CLUSTER_NAME.yaml
+      ```
+     Example command output
+      ```
+      Performing setup and validations
+      ✅ validation succeeded {"validation": "docker Provider setup is valid"}
+      Creating new bootstrap cluster
+      Installing cluster-api providers on bootstrap cluster
+      Provider specific setup
+      Creating new workload cluster
+      Installing networking on workload cluster
+      Installing cluster-api providers on workload cluster
+      Moving cluster management from bootstrap to workload cluster
+      Installing EKS-A custom components (CRD and controller) on workload cluster
+      Creating EKS-A CRDs instances on workload cluster
+      Installing GitOps Toolkit on workload cluster
+      GitOps field not specified, bootstrap flux skipped
+      Deleting bootstrap cluster
+      🎉 Cluster created!
+      ----------------------------------------------------------------------------------
+      The Amazon EKS Anywhere Curated Packages are only available to customers with the
+      Amazon EKS Anywhere Enterprise Subscription
+      ----------------------------------------------------------------------------------
+      Installing curated packages controller on management cluster
+      secret/aws-secret created
+      job.batch/eksa-auth-refresher created
+      ```
+   - Cluster creation with optional curated packages
+
+     {{% alert title="Note" color="primary" %}}
+   * It is *optional* to install curated packages as part of the cluster creation.
+   * `eksctl anywhere version` version should be later than `v0.9.0`.
+   * Post-creation installation and detailed package configurations can be found [here.]({{< relref "../../tasks/packages" >}})
+   * The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/)
+     {{% /alert %}}
```

Comment:
Can we move all this to a step before the cluster creation. There should be a step of configuring this stuff the way the prod version are documented

Reply:
There already was documentation looks like. I added a small note.